test/strings/search.jl

```diff
@@ -390,6 +390,36 @@ s_18109 = "fooα🐨βcd3"
     @test findall("aa", "aaaaaa", overlap=true) == [1:2, 2:3, 3:4, 4:5, 5:6]
 end
 
+# issue 37280
+@testset "UInt8, Int8 vector" begin
+    for VT in [Int8, UInt8]
+        A = VT[0x40, 0x52, 0x62, 0x52, 0x62]
```

Comment:
This only tests the case where the pattern and array have the same eltype. It would be nice to test when they have different types, using
```suggestion
    for T in [Int8, UInt8], VT in [Int8, UInt8]
        A = T[0x40, 0x52, 0x62, 0x52, 0x62]
```